<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productAdder</class>
 <widget class="QWidget" name="productAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить заказ</string>
  </property>
  <widget class="QPushButton" name="cancelEditorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="takeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Вы можете изменить адрес доставки&lt;br/&gt;и время готовности или доставки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="deliveryTake">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Доставка</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="attendanceTake">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Самовывоз</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Город:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="citySelector">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Красноярск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Дивногорск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сосновоборск</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="streetLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Улица:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="streetInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Академика Киренского</string>
   </property>
  </widget>
  <widget class="QLabel" name="houseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дом/стр:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="houseInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>26Б</string>
   </property>
  </widget>
  <widget class="QLabel" name="aptLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кв/офис:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aptInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editOrderButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить изменения</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="timeToDeliver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время готовности/доставки:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
